bundles/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/handler/CBusLightHandler.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.cbus.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.IncreaseDecreaseType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.cbus.CBusBindingConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.daveoxley.cbus.CGateException;
+import com.daveoxley.cbus.Group;
+
+/**
+ * The {@link CBusLightHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Scott Linton - Initial contribution
+ */
+@NonNullByDefault
+public class CBusLightHandler extends CBusGroupHandler {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
I fixed this but wanted to leave CBusGroupHandler using CBusLight... , CBusTemperature.. loggers. So i passed that down in the constructor and they all use the logger from the base class. Let me know if you agree this was an appropriate way to handle this.

Reply:
I understand that you are trying to make the parent and children use the same logger but I would discourage this since openhab's log format only outputs the logger name, not the class that a log was actually issued from. So it would make tracking down the source of log statements rather difficult. Each class should have its own logger to make things easier to debug.

Reply:
I have updated so there is now a different logger for the base class.